packages/mui-material/src/Chip/Chip.test.js

```diff
@@ -30,6 +30,14 @@ describe('<Chip />', () => {
     refInstanceof: window.HTMLDivElement,
     testComponentPropWith: 'span',
     skip: ['componentsProp'],
+    slots: {
+      root: {
+        expectedClassName: classes.root,
+      },
+      label: {
+        expectedClassName: classes.label,
+      },
+    },
```

Comment:
Given the comment about the `shouldForwardComponentProp`. I think we need to add more tests.

- With `clickable: true` and a custom `component` prop.
- With `onDelete` and a custom `component` prop.

Reply:
https://github.com/mui/material-ui/blob/4cfd2f73c291df2a57fa5feb6bffdd39c544d838/packages/mui-material/src/Chip/Chip.test.js#L83-L88 There is test which tests with same props combination as mentioned. is this test sufficient or do you have any other test in mind?